src/main/java/frc/robot/subsystems/AddressableLEDSubsystem.java

```diff
@@ -0,0 +1,137 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.subsystems;
+
+import edu.wpi.first.wpilibj.AddressableLED;
+import edu.wpi.first.wpilibj.AddressableLEDBuffer;
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import static frc.robot.Constants.LEDConstants.*;
+
+public class AddressableLEDSubsystem extends SubsystemBase {
+  /** Creates a new AddressableLEDSubsystem. */
+
+  AddressableLED LED;
+  AddressableLEDBuffer LEDBuffer;
```

Comment:
is it possible to make these private & final?

additionally, variable names with lowercase first letters are preferred for style coherence. i understand this one is awkward because its an acronym but this could become `led`, `ledBuffer`, etc if we decide it's better